ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -260,11 +241,13 @@ Item {
             messageId: model.id
             communityId: model.communityId
             responseToMessageWithId: model.responseToMessageWithId
+            responseToMessage: model.responseToMessageWithId && model.responseToMessage ? JSON.parse(model.responseToMessage) : null
```

Comment:
I already fix that in my PR, but in case yours gets merged first, we need a try/catch in case the `responseToMessage` is not valid JSON